pcawg_metadata_parser/run_me.sh

```diff
@@ -31,6 +31,11 @@ cd ../s3-transfer-operations/
 git pull
 cd $DIR
 
+echo update ceph-transfer-operations git submodule
+cd ../ceph_transfer_ops/
+git pull
```

Comment:
please make sure this new git submodule is set up and initiated on the pancancer.info server, otherwise, it will fail.